src/middlewared/middlewared/plugins/zfs_/snapshot.py

```diff
@@ -300,4 +302,6 @@ def do_delete(self, id, options):
 
             raise CallError(str(e))
         else:
+            # TODO: Events won't be sent for child snapshots in recursive delete
+            self.middleware.send_event(f'{self._config.namespace}.query', 'REMOVED', id=id)
```

Comment:
@yocalebo should I create an issue for UI team so they handle that on their side? (they already have a list of snapshots when they receive this event)

Reply:
@themylogin yes, please

Reply:
https://ixsystems.atlassian.net/browse/NAS-123278